.circleci/config.yml

```diff
@@ -182,7 +182,7 @@ workflows:
           - test_optimus_mouse
           - test_smartseq2
           - test_smartseq2_single_end
-          - test_emptyDropsWrapper
+#          - test_emptyDropsWrapper
```

Comment:
I think it would be better to delete this and create a JIRA in our backlog to track adding this test back.

Reply:
GL-854 created